docs/ai/conceptual/zero-shot-learning.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Zero-shot and few-shot learning"
+description: "Learn the use cases for zero-shot and few-shot learning in prompt engineering."
+author: catbutler
+ms.author: [your Microsoft alias or a team alias]
+ms.service: [the approved service name]
+ms.topic: concept-article #Don't change.
+ms.date: [mm/dd/yyyy]
+
+#customer intent: As a .NET developer, I want to understand how zero-shot and few-shot learning techniques can help me improve my prompt engineering.
+
+---
+
+# Zero-shot and few-shot learning
+
+This article explains zero-shot learning and few-shot learning for prompt engineering in .NET, including their primary use cases.
+
+GPT model performance benefits from [prompt engineering](prompt-engineering-in-dot-net.md), the practice of providing instructions and examples to a model to refine its output. Zero-shot learning and few-shot learning are techniques that you can use when providing examples.
+
+With zero-shot learning, you include prompts but not verbatim completions (you can include completions that only consist of cues). Zero-shot learning relies entirely on the model's existing knowledge to generate responses, reducing the amount of tokens created, which can help you control costs. However, zero-shot learning doesn't add to the model's knowledge.
+
+Here's an example zero-shot prompt that tells the model to evaluate user input to determine which of four possible intents the input represents, and then to preface its response with **"Intent: "**.
+
+```csharp
+prompt = @$"Instructions: What is the intent of this request?
+If you don't know the intent, don't guess; instead respond with ""Unknown"".
+Choices: SendEmail, SendMessage, CompleteTask, CreateDocument, Unknown.
+User Input: {request}
+Intent: ";
```

Comment:
Here, this does use string interpolation, but I'd still use raw string literals to improve formatting:

```suggestion
prompt = $"""
Instructions: What is the intent of this request?
If you don't know the intent, don't guess; instead respond with "Unknown".
Choices: SendEmail, SendMessage, CompleteTask, CreateDocument, Unknown.
User Input: {request}
Intent: 
""";
```